software/wmla120.py

```diff
@@ -283,6 +283,15 @@ def README(self):
                 'Note: The \'pup\' cli supports tab autocompletion.\n\n')
         print(text)
 
+    def _is_nginx_running(self):
```

Comment:
This function is a little confusing in that it will return either `True` or `None`. The `None` class `__bool__()` method does return `False`, so for example `bool(_is_nginx_running()) == False` evaluates to `True` while `_is_nginx_running() == False` evaluates to `False`. I think it's worth throwing in a `return False` at the end of the function. to provide a consistent return type.

Reply:
Agreed